ca/certificates.go

```diff
@@ -209,23 +200,60 @@ func (rca *RootCA) RequestAndSaveNewCertificates(ctx context.Context, paths Cert
 		return nil, err
 	}
 
-	// Ensure directory exists
-	err = os.MkdirAll(filepath.Dir(paths.Cert), 0755)
+	kekUpdate, err := rca.getKEKUpdate(ctx, X509Cert, tlsKeyPair, r)
```

Comment:
Suggest looping here, like the `for i := 0; i != 5; i++ {` above, so that if the manager it first tries to contact is unreachable, it tries again.